docker-compose.yml.dist

```diff
@@ -40,6 +40,8 @@ services:
 
 networks:
   symfony-app-network:
+    name: symfony-app-network
+    external: true
```

Comment:
`external: true` is only needed if you are planning to see these services outside this network. As this is not a requirement for all project maybe you can remove this

Reply:
If this is not a requirement, we should also remove the creation of an external network in the `start` and `build`  rules to delegate the creation of the network to _docker compose_(It will create a network called `<name_of_the_directory>_<name_of_the_network>` 🤔

So, what is the expected behavior? _docker compose_ create the network or we do it in some rules of our Makefile? Or both?

Reply:
The idea behind creating a named network is to make easier to find it using docker commands, otherwise docker creates a "random" name for it that is a bit difficult to track if you have many networks. I think you can keep the network name without the external flag

Reply:
That's a good idea but if we create the network in the `make start` or `make build` rule, then, if we have in the `networks` configuration only the `name` this error will be thrown when we try to start our containers:
<img width="854" alt="image" src="https://github.com/codenip-tech/symfony-base-repository/assets/62360034/c8d83b7b-3d9c-46ec-b3bb-3a28412d9e43">

With the `external` config as you said, we use the network created in our Makefile rules, so, if we don't want this external network by default, we should remove the `external` configuration in `docker-compose.yml` and the creation of the external network in `make start` and `make build`

Reply:
But that's why the `|| true` is there. If the network exists it will continue the execution. My point here is that by default we don't need to create the network as external and if we do so, once you do `make start` the network will be created as external and if you don't want that, you have to remove all the containers, remove it and then update `docker-compose.yml` to remove the external flag.